src/_includes/partials/template-list.njk

```diff
@@ -0,0 +1,27 @@
+{% set templates = [cardlist.type, locale | upper] | join %}
+{% set templateCards = collections.all | eleventyNavigation(templates) %}
+
+{% if cardlist.type === 'pageTemplates' and cardlist.state === 'published' %}
+  <div class="my-500">
+    <gcds-grid tag="ul" columns="1fr" columns-tablet="1fr 1fr" columns-desktop="1fr 1fr 1fr" gap="400">
```

Comment:
We can remove `gap=400` here, it's an old leftover when we still had a gap property.

Reply:
```suggestion
    <gcds-grid tag="ul" columns="1fr" columns-tablet="1fr 1fr" columns-desktop="1fr 1fr 1fr">
```

Reply:
Done!